apollo-portal/src/main/resources/static/i18n/zh-CN.json

```diff
@@ -187,6 +187,8 @@
   "Component.Namespace.Master.Items.FilterItem": "过滤配置",
   "Component.Namespace.Master.Items.SyncItemTips": "同步各环境间配置",
   "Component.Namespace.Master.Items.SyncItem": "同步配置",
+  "Component.Namespace.Master.Items.RevokeItemTips": "撤销配置的修改",
```

Comment:
please add the English translations to en.json

Reply:
okay